src/update-project.ts

```diff
@@ -318,6 +318,7 @@ export async function run(): Promise<void> {
   );
   if (Object.entries(projectMetadata).length === 0) return;
 
+  setOutput("field_read_value", contentMetadata.field.value);
```

Comment:
🤔 Should line 328/9 (the `else`) be `field_read_value`? Alternatively, should the `else` be removed if we always output the read value?

Reply:
I think it's fine as is (I afaik it's the same behavior as in the bash implementation):

- `field_read_value` is always set with the value before the update (in both `read` and `update`)
- `field_updated_value` is the same as `field_read_value` in case of `read`, and is the updated value in case of `update`

The way I see it, `if/else` is still needed.